bom-weekly/pom.xml

```diff
@@ -219,6 +229,11 @@
                 <artifactId>theme-manager</artifactId>
                 <version>1.5</version>
             </dependency>
+            <dependency>
+                <groupId>io.jenkins.plugins</groupId>
+                <artifactId>warnings-ng</artifactId>
+                <version>9.15.0</version>
+            </dependency>
```

Comment:
`warnings-ng` failed in [this run](https://ci.jenkins.io/job/Tools/job/bom/job/PR-1384/1/) with:

```
[ERROR] Error resolving version for plugin 'org.jenkins-ci.tools:maven-hpi-plugin' from the repositories [local (/home/jenkins/.m2/repository), central (https://repo.maven.apache.org/maven2)]: Plugin not found in any plugin repository -> [Help 1]
org.apache.maven.plugin.version.PluginVersionResolutionException: Error resolving version for plugin 'org.jenkins-ci.tools:maven-hpi-plugin' from the repositories [local (/home/jenkins/.m2/repository), central (https://repo.maven.apache.org/maven2)]: Plugin not found in any plugin repository
```